DXMainClient/DXGUI/Generic/MainMenu.cs

```diff
@@ -696,7 +694,30 @@ private void BtnOptions_LeftClick(object sender, EventArgs e)
 
         private void BtnNewCampaign_LeftClick(object sender, EventArgs e)
         {
-            innerPanel.Show(innerPanel.CampaignSelector);
+            if (ClientConfiguration.Instance.LocalGame == ProgramConstants.YR)
+            {
+                if (File.Exists(ProgramConstants.GamePath + ClientConfiguration.Instance.CampaignPath))
+                {
+                    LaunchCampaignExe();
+                }
+                else
+                {
+                    innerPanel.Show(null); // Darkening
+
+                    XNAMessageBox msgBox = new XNAMessageBox(WindowManager,
+                        "Failed to launch",
+                        "Could not find the campaign, please contact support.",
+                        XNAMessageBoxButtons.OK
+                    );
+
+                    msgBox.OKClickedAction = MsgBox_OKClicked;
+                    msgBox.Show();
```

Comment:
Since you're using the message box only for displaying a single message and don't need it afterwards, use `XNAMessageBox.Show` instead of creating it manually.